lib/outputmanager.js

```diff
@@ -77,18 +77,17 @@ var Constants = {
         Less: 'less',
         Framework: 'adapt_framework',
         Plugins: 'plugins',
-        ThemeVariablesDirectory: 'yyyyy',
         CustomStyleDirectory: 'zzzzz'
     },
     Filenames: {
       Download: 'download.zip',
       Main: 'index.html',
       Rebuild: '.rebuild',
-      CustomStyle: 'zzzzz.less',
+      CustomStyle: '2-customLess.less',
       Bower: 'bower.json',
       Package: 'package.json',
       Metadata: 'metadata.json',
-      ThemeVariables: 'yyyyy.less',
+      ThemeVariables: '1-themeVariables.less',
```

Comment:
I don't think `Constants.Filenames.Variables` is used any longer so could switch to that instead of having `ThemeVariables`.